src/utils/Processing.ts

```diff
@@ -240,6 +241,26 @@ export function fillTank(rawFSHes: RawFSH[], config: Configuration): FSHTank {
   return new FSHTank(docs, config);
 }
 
+function checkNullValuesOnArray(resource: any): void {
+  const resourceType = resource instanceof InstanceDefinition ? 'Instance' : resource.resourceType;
+  const resourceName =
+    resource instanceof InstanceDefinition ? resource._instanceMeta.name : resource.title;
+  for (const property in resource) {
```

Comment:
It looks like you're only iterating top-level fields.  I think we want to detect missing values in arrays that are at nested paths too.  For example:
```json
{
  "foo": {
    "bar": [ "a", null, "c"]
  }
}
```